code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -201,6 +201,7 @@
 		return 0
 
 /mob/living/simple_animal/hostile/proc/on_attack_mob(var/mob/hit_mob, var/obj/item/organ/external/limb)
+	limb.add_autopsy_data("Mauling by [src.name]")
```

Comment:
```suggestion
	if(QDELETED(limb))
		return

	limb.add_autopsy_data("Mauling by [src.name]")
```
Have to check that the limb still exists, otherwise it will runtime